CHANGES.rst

```diff
@@ -84,6 +84,11 @@ Fixed:
   and importlib_metadata 3.6)
 * Handle deprecation of distutils in Python 3.10 (use the packaging package)
 
+Documentation:
```

Comment:
I normally credit PR authors in the changelog. Since you'll be make most of the changes, we could change this to:

    Documentation (PRs #330, #331 by Daniele Procida)

I can add the PR numbers after merging them.